spec/openapi.yaml

```diff
@@ -10342,6 +10326,92 @@ components:
           description: any text to describe the reason the file is canceled
           type: string
       type: object
+    JobBatchCreateJobRequest:
```

Comment:
I object to continuing the ridiculous practice of extending `components` in the main file but only using them in a specific API. Let's add to our file
```yaml
components:
  responses:
```